Makefile

```diff
@@ -0,0 +1,21 @@
+PACKAGE=rest-dbus
+
+prefix?=/usr/local
+bin=$(prefix)/usr/bin
```

Comment:
Shouldn't bin and share drop the /usr?  /usr/local/bin would be the ideal default path and not /usr/local/usr/bin ?